docs/attributes.md

```diff
@@ -0,0 +1,47 @@
+# Object Attributes (attrs)
+
+# This is still a work in progress (mistakes are common).
+
+## For objects that have associated events
+> Required in **bold**
+- duration: number; The number of users that can be admitted in the event associated to the event.
+- eventStartDelay: The number of minutes before the the event starts after the startAt field of the event.
```

Comment:
it is actually the time between the end of registration and the beginning of the event (= it's creation)

Reply:
Oh ok